File: 240110/함수를 이용한 합과 소수 판별/use-functions-to-determine-sums-and-decimals.java
import java.util.Scanner;
public class Main {
    public static boolean sosu(int a){
        
            for(int j =2; j<a; j++){
                if(a%j == 0 ){
                    return false;
                }
            }
            return true;
        
    }
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int cnt = 0 ;

        for(int i = a; i<=b; i++){
            if(sosu(i)){
                if (((i/10) + (i%10)) %2 == 0){
                    cnt++;
                }
            }
        }
        System.out.print(cnt);

    }
}

File: 240110/함수를 이용한 합과 소수 판별/use-functions-to-determine-sums-and-decimals.py
a,b = map(int,input().split())

def sosu(i):
    for  k in range(2,i):
        if i% k == 0:
            return False
    return True

cnt = 0 
for i in range(a,b+1):
    if sosu(i):
        if ((i//10) + (i%10) ) %2 == 0:
            cnt+=1
print(cnt)